x-pack/qa/rolling-upgrade/build.gradle

```diff
@@ -188,13 +190,15 @@ for (Version version : bwcVersions.wireCompatible) {
             setting 'xpack.security.transport.ssl.enabled', 'true'
             setting 'xpack.security.authc.token.timeout', '60m'
             setting 'logger.org.elasticsearch.xpack.security.authc.TokenService', 'trace'
-            if (project.inFipsJvm) {
-                setting 'xpack.security.transport.ssl.key', 'testnode.pem'
-                setting 'xpack.security.transport.ssl.certificate', 'testnode.crt'
-                keystoreSetting 'xpack.security.transport.ssl.secure_key_passphrase', 'testnode'
-            } else {
-                setting 'xpack.security.transport.ssl.keystore.path', 'testnode.jks'
-                setting 'xpack.security.transport.ssl.keystore.password', 'testnode'
+            rootProject.globalInfo.ready {
```

Comment:
I think you'll be able to get rid of ready with "${-> }" here, and with that,  we wouldn't need `ready` at all

Reply:
The issue here is not that the system property _values_ are different. The set of system properties themselves look to be different depending on FIPS or not.